recipe/build.sh

```diff
@@ -218,7 +218,6 @@ if [[ "$PKG_NAME" == "libtorch" ]]; then
   for f in ATen caffe2 tensorpipe torch c10; do
     mv torch/include/$f ${PREFIX}/include/$f
   done
-  rm ${PREFIX}/lib/libtorch_python.*
```

Comment:
This won't work AFAIU; `libtorch_python.so` depends on a specific python version, while the conda package `libtorch` intentionally does not depend on python at all.

If we want to ship `libtorch_python.so`, we should most likely package it in the `pytorch` output (or perhaps create a separate output).

Reply:
Thank you for pointing out that `libtorch` should not depend on Python. Moving it or creating a separate output seems like a viable solution.

I am relatively new to the build script of this feedstock and find it more complex than other packages I've worked with. Could you please provide some guidance on how to implement your suggestion?

Reply:
> Could you please provide some guidance on how to implement your suggestion?

As a first approach, it should work to "install" `libtorch_python.so` in `build_pytorch.sh`, by copying from the build cache (that's created & populated during the execution of `build.sh`) into `$PREFIX/lib`.